content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -120,95 +134,51 @@ using the (deprecated) v1alpha2 API instead.
-overlay
-br_netfilter
-EOF
-
-sudo modprobe overlay
-sudo modprobe br_netfilter
-
-# Setup required sysctl params, these persist across reboots.
-cat <<EOF | sudo tee /etc/sysctl.d/99-kubernetes-cri.conf
-net.bridge.bridge-nf-call-iptables  = 1
-net.ipv4.ip_forward                 = 1
-net.bridge.bridge-nf-call-ip6tables = 1
-EOF
-
-# Apply sysctl params without reboot
-sudo sysctl --system
-```
-
-Install containerd:
-
-{{< tabs name="tab-cri-containerd-installation" >}}
-{{% tab name="Linux" %}}
-
-1. Install the `containerd.io` package from the official Docker repositories. 
-Instructions for setting up the Docker repository for your respective Linux distribution and 
-installing the `containerd.io` package can be found at 
-[Install Docker Engine](https://docs.docker.com/engine/install/#server).
-
-2. Configure containerd:
+   (these instructions apply to Linux nodes only)
 
    ```shell
-   sudo mkdir -p /etc/containerd
-   containerd config default | sudo tee /etc/containerd/config.toml
+   cat <<EOF | sudo tee /etc/modules-load.d/containerd.conf
+   overlay
+   br_netfilter
+   EOF
+
+   sudo modprobe overlay
+   sudo modprobe br_netfilter
+
+   # Setup required sysctl params, these persist across reboots.
+   cat <<EOF | sudo tee /etc/sysctl.d/99-kubernetes-cri.conf
+   net.bridge.bridge-nf-call-iptables  = 1
+   net.ipv4.ip_forward                 = 1
+   net.bridge.bridge-nf-call-ip6tables = 1
+   EOF
+
+   # Apply sysctl params without reboot
+   sudo sysctl --system
    ```
 
-3. Restart containerd:
+1. Install containerd:
 
-   ```shell
-   sudo systemctl restart containerd
-   ```
-
-{{% /tab %}}
-{{% tab name="Windows (PowerShell)" %}}
-
-Start a Powershell session, set `$Version` to the desired version (ex: `$Version="1.4.3"`), 
-and then run the following commands:
-
-1. Download containerd:
-
-   ```powershell
-   curl.exe -L https://github.com/containerd/containerd/releases/download/v$Version/containerd-$Version-windows-amd64.tar.gz -o containerd-windows-amd64.tar.gz
-   tar.exe xvf .\containerd-windows-amd64.tar.gz
-   ```
-
-2. Extract and configure:
+   Visit
+   [Getting started with containerd](https://containerd.io/docs/getting-started/#starting-containerd)
+   and follow the instructions there, up to the point where you have a valid
+   configuration file (on Linux: `/etc/containerd/config.toml`).
 
+   If you are running Windows, you might want to exclude containerd from Windows Defender Scans
    ```powershell
-   Copy-Item -Path ".\bin\" -Destination "$Env:ProgramFiles\containerd" -Recurse -Force
-   cd $Env:ProgramFiles\containerd\
-   .\containerd.exe config default | Out-File config.toml -Encoding ascii
-
-   # Review the configuration. Depending on setup you may want to adjust:
-   # - the sandbox_image (Kubernetes pause image)
-   # - cni bin_dir and conf_dir locations
-   Get-Content config.toml
-
-   # (Optional - but highly recommended) Exclude containerd from Windows Defender Scans
```

Comment:
Why drop these lines?

Reply:
https://kubernetes.io/docs/contribute/style/content-guide/#what-s-allowed basically. We should aim to link to external docs; I'm also happy to see work done on those external docs (and maybe I will do some of that myself).